cli.py

```diff
@@ -229,12 +206,23 @@ def buildout(self):
             subprocess.check_call([sys.executable, 'bootstrap.py'])
             subprocess.check_call(['bin/buildout'])
 
+    def setup(self):
+        with FSFolderContext(self.generated_package_path):
+            logger.info(
+                "Run setup.py for package {}".format(os.getcwd()))
+
+            subprocess.check_call(['python3', '-m', 'venv', '.'])
+            subprocess.check_call(['bin/pip', 'install', '.[dev]', '-cversions.txt'])
+            subprocess.check_call(['bin/pip', 'install', '.[tests]', '-cversions.txt'])
+
     def run_template_tests(self):
         with FSFolderContext(self.generated_package_path):
             logger.info(
                 "Run tests for package {}".format(os.getcwd()))
-
-            subprocess.check_call(['bin/test'])
+            if self.template_name != 'django':
+                subprocess.check_call(['bin/test'])
+            else:
+                subprocess.check_call(['bin/tox'])
```

Comment:
I think I don't understand this correctly: the template does not create a `bin/tox`, does it?
I'd probably change this check to:
if `tox.in` exists => run `tox`
else => run `bin/test`